snuba/web/views.py

```diff
@@ -225,6 +227,7 @@ def health():
 def parse_request_body(http_request):
     with sentry_sdk.start_span(description="parse_request_body", op="parse"):
         try:
+            metrics.timing("http_request_body_length", len(http_request.data))
```

Comment:
I think `http_request.data` is a string (not bytes, at least per the [documentation](https://flask.palletsprojects.com/en/1.1.x/api/#flask.Request.data) — [`request` comes from here](https://github.com/pallets/flask/blob/64ba43411fbfb2e435455b0df2453d7809b7c5a8/src/flask/globals.py#L60) so I pretty much gave up immediately on trying to trace the actual source down) so this is in _characters_ and not bytes (though they should be non-multibyte characters because JSON escaping.)

I don't trust the `Content-Length` header enough to use it here, either.

Reply:
I'd also love to log the dataset here but the majority of our queries still come through [`unqualified_query_view`](https://github.com/getsentry/snuba/blob/8a3bf86ce4e2df7cda5b39e6bf5e01fb87a86514/snuba/web/views.py#L235) to my knowledge so I'm not sure that it's worth it in the case that the JSON is malformed.

Speaking of malformed JSON, I guess I should also move this out of the `try`/`except` block.

Reply:
Yeah, I think that's fine. The goal is just to get a high watermark anyways.